aspnetcore/data/ef-rp/read-related-data.md

```diff
@@ -91,93 +122,98 @@ Run the app and select the **Courses** tab to see the list with department names
 
 ![Courses Index page](read-related-data/_static/courses-index.png)
 
-## Create an Instructors page that shows Courses and Enrollments
+<a name="select"></a>
+### Loading related data with Select
 
-In this section, you'll create a controller and view for the Instructor entity in order to display the Instructors page:
+The `OnGetAsync` method loads related data with the `Include` method:
 
-![Instructors Index page](read-related-data/_static/instructors-index.png)
+[!code-csharp[Main](intro/samples/cu/Pages/Courses/Index.cshtml.cs?name=snippet_RevisedIndexMethod&highlight=4)]
 
-This page reads and displays related data in the following ways:
+The `Select` operator loads only the related data needed, and is generally faster than using `Include`. For single items, like the `Department.Name` it uses a SQL INNER JOIN. For collections it uses another database access, but so does the .`Include` operator on collections.
```

Comment:
This seems to contradict the single join statements I commented on above. Also, Include may also use an inner join depending on the configuration of the relationships.